resources/zib/zib-LaboratoryTestResult.xml

```diff
@@ -0,0 +1,358 @@
+      <path value="Observation.interpretation.coding" />
+      <max value="0" />
+    </element>
+    <element id="Observation.interpretation:resultInterpretation.text">
+      <path value="Observation.interpretation.text" />
+      <short value="ResultInterpretation" />
+      <definition value="Comment of the laboratory specialist regarding the interpretation of the results" />
+      <alias value="UitslagInterpretatie" />
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.32" />
+        <comment value="ResultInterpretation" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <max value="1" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Comments, such as a textual interpretation or advice accompanying the result, for example." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.5" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.method">
+      <path value="Observation.method" />
+      <short value="TestMethod" />
+      <definition value="The test method used to obtain the result." />
+      <alias value="Testmethode" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-laboratorytestresult-v4.6-2020EN" />
+        <map value="NL-CM:13.1.9" />
+        <comment value="TestMethod" />
+      </mapping>
+    </element>
+    <element id="Observation.specimen">
+      <path value="Observation.specimen" />
+      <comment value="The Specimen concept of the zib defines either the collected source material without further processing, or a microorganism, optionally including the source material from which it whas isolated.&#xD;&#xA;&#xD;&#xA;* If the Specimen pertains a material (zib concept SpecimenMaterial), the `Specimen.type` of the resource referenced here should contain a coding indicating a specimen material.&#xD;&#xA;* If the Specimen pertains a microorganism (zib concept Microorganism), the `Specimen.type` of the resource referenced here should contain a coding indicating the microorganims. If required, the sample material from which the microorganism was isolated can be referenced using `Specimen.parent` on that resource, but it is not referenced here." />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Specimen" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-LaboratoryTestResult.Specimen" />
+      </type>
+    </element>

```

Comment:
Missing the following info:

* Mapping
* Short
* Alias
* Definition? Not sure about this

Reply:
The Specimen container is represented using profile zib-LaboratoryTestResult.Specimen. This is merely the way to link the two. There's documentation on how to do so.